deploy/etc/worker-types.json

```diff
@@ -1,5 +1,7 @@
 {
   "hvm-builder": [
+    "android-components-g",
+    "application-services-r",
```

Comment:
Based on [this report](https://bugzilla.mozilla.org/show_bug.cgi?id=1535352#c0) I suspect we should add some more that just these, but this is a good start for [bug 1535179](https://bugzil.la/1535179), and we can probably add the others in [bug 1535352](https://bugzil.la/1535352).

Reply:
I think @tomprince is adding a bunch more.